packages/orm/mongoose/src/decorators/postHook.ts

```
@@ -46,10 +46,10 @@ import {schemaOptions} from "../utils/schemaOptions.js";
  * @mongoose
  * @class
  */
-export function PostHook<T = any>(method: string, fn: MongoosePostHookCB<T>): ClassDecorator;
-export function PostHook<T = any>(method: string, fn: MongoosePostHookCB<T>, options: MongooseHookOptions): ClassDecorator;
-export function PostHook<T = any>(method: string, options: MongooseHookOptions): StaticMethodDecorator;
-export function PostHook<T = any>(method: string, ...params: any[]): Function {
+export function PostHook<T = any>(method: MongooseMethods, fn: MongoosePostHookCB<T>): ClassDecorator;
+export function PostHook<T = any>(method: MongooseMethods, fn: MongoosePostHookCB<T>, options: MongooseHookOptions): ClassDecorator;
+export function PostHook<T = any>(method: MongooseMethods, options: MongooseHookOptions): StaticMethodDecorator;
+export function PostHook<T = any>(method: MongooseMethods, ...params: any[]): Function {
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)